topics/sequence-analysis/tutorials/ncbi-fcs/tutorial.md

```diff
@@ -0,0 +1,193 @@
+>    ```
+>    https://zenodo.org/records/10881551/files/FCS.combo.test.fa
+>    ```
+>
+> **Step 3:** Upload `fasta.gz` dataset into Galaxy
+>    > {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    > {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+{: .hands_on}
+
+## Confirm dataset upload
+
+Your data should look like this:
+
+![Galaxy History with Uploaded Data](../../images/ncbi-fcs/fcs-galaxy-1.jpg "Figure 1. Galaxy History with Uploaded Data.")
+
+# Importing Workflows
+
+Next we will import a Galaxy workflow - a chain of tools to perform a set of operations on a user-supplied input. Specificially in this workflow, we will:
+
+1. Screen the genome for foreign organism sequences using **FCS-GX** `screen` mode.
+1. Produce a cleaned set of contigs using **FCS-GX** `clean` mode.
+1. Screen the genome for synthetic sequences using **FCS-adaptor** and remove identified contaminants. 
+1. Produce a final set of cleaned contigs using a second iteration of **FCS-GX** `clean` mode.
+
+![NCBI Foreign Contamination Screen Galaxy Workflow](../../images/ncbi-fcs/fcs-galaxy-2.jpg "Figure 2. NCBI Foreign Contamination Screen Galaxy Workflow.")
+
+> <hands-on-title> Importing Galaxy Workflows </hands-on-title>
+>
+> **Step 1:** Copy the workflow URLs into clipboard.
+>
+>    ```
+>    https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/ncbi-fcs/workflows/ncbi-fcs.ga
+>    ```
+>
+> **Step 2:** Open Galaxy.
+> 
+>    > <warning-title>Log in to Galaxy</warning-title>
+>    > Make sure you are logged in to your Galaxy account!
+>    {: .warning}
+>
+>
+> **Step 3:** Import NCBI Foreign Contamination Screen Workflow into your Galaxy interface.
+>    > {% snippet  faqs/galaxy/workflows_import.md %}
+>
+{: .hands_on}
+
+Once the workflow is loaded, your workflow page should look like this:
+
+![Galaxy Workflows page with NCBI Foreign Contamination Screen](../../images/ncbi-fcs/fcs-galaxy-3.jpg "Figure 3. Galaxy Workflows page with NCBI Foreign Contamination Screen.")
+
+# Contamination screening
+
+## Running the NCBI FCS Workflow
+
+Next you will run the Galaxy workflow. Here you will configure the parameters for the FCS run according to the target organism.
+
+> <hands-on-title> Running the NBCI FCS Workflow </hands-on-title>
+> **Step 1:** Collect inputs
+> 1. Assembled genome in `fasta/fasta.gz` format
+> 1. Taxonomic information for source organism
+>
+> **Step 2:** Launch the Workflow
+> 1. Click the Workflow menu located in the top menu bar
+> 1. Click the {% icon workflow-run %} button located in the NCBI Foreign Contamination Screen Workflow box
+> 1. Click "Expand to full workflow form." 
+> 1. In the {% icon param-file %} 1: Input genome menu: 
+> - {% icon param-file %} *"Input file (Fasta file)"*: `1:FCS.combo.test.fa`
+> 1. In the {% tool [2: FCS-GX screen](toolshed.g2.bx.psu.edu/repos/iuc/ncbi_fcs_gx/ncbi_fcs_gx/0.5.0+galaxy0) %} menu:
```

Comment:
```suggestion
> 1. In the **2: FCS-GX screen** menu:
```

I think tool links are not necessary here, and might be a distraction since they aren't running that specific tool.